dev-requirements.txt

```diff
@@ -1,6 +1,6 @@
 freezegun
 pretend
-pytest
+pytest<=3.2.5
```

Comment:
Just `< 3.3.0` would be kinda more correct but it has no actual practical difference unless pytest has another release in the 3.2 series.

Reply:
More prudent maybe? :P

Reply:
Indeed.

TIL - prudent. :)